### PR TITLE
docs(plugins/syslog) docs for facility option

### PR DESCRIPTION
**Pull request by locao**

### CODE REVIEW

File: app/_hub/kong-inc/syslog/index.md [hunk 1]
```diff
@@ -90,6 +91,15 @@ params:
         A list of key-value pairs, where the key is the name of a log field and
         the value is a chunk of Lua code, whose return value sets or replaces
         the log field value.
+    - name: facility
+      required: false
+      default: "`user`"
+      datatype: string
+      description: |
+        The facility is used by the operating system to decide how to handle each log message. This
```
Comment (HCloward) [inline app/_hub/kong-inc/syslog/index.md:99]: Should we include a description of what each of these means? Also, what does "how to handle each log message" mean? What exactly is done here? @dndx suggested this is a way for the Syslog plugin to group messages from different sources - if so I would mention that here.

**Review by HCloward — COMMENTED** [review]
I am not sure the description is very clear. What is the value of using this parameter?

File: app/_hub/kong-inc/syslog/index.md (hunk 1 above)
Comment (dndx) [reply app/_hub/kong-inc/syslog/index.md:99]: Yes that's what it does. See: https://en.wikipedia.org/wiki/Syslog#Facility

Comment (HCloward) [reply app/_hub/kong-inc/syslog/index.md:99]: ok, that's fine with me

**Review by HCloward — APPROVED** [review]

**Pull request merged**